LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2019 Rhythmic Technologies, Inc.
```

Comment:
```suggestion
Copyright (c) 2020 Rhythmic Technologies, Inc.
```

Reply:
Ultimately this is simpler only because it's less flexible, but it's more specialized to our needs.